doc/content/tutorial/tutorial.md

```diff
@@ -0,0 +1,89 @@
+
+!listing tutorial/site_response_1.i
+         start=Variables
+         end=AuxVariables
+
+The next block of text is the Variables block. This is where the user defines the non-linear variables being solved for, such as displacements, temperature, etc. See [Variables](syntax/Variables/index.md) for more information on the variables system.
+
+!listing tutorial/site_response_1.i
+         start=AuxVariables
+         end=Kernels
+
+Following Variables, the user must then define the AuxVariables. Auxiliary variables are values which can be derived from the primary variables or explicitly defined, this includes velocity, acceleration, stress etc. See [AuxVariables](syntax/AuxVariables/index.md) for more information on the AuxVariables system.
+
+!listing tutorial/site_response_1.i
+         start=Kernels
+         end=AuxKernels
+
+The Kernels block is where the physics of interest is defined. Because MASTODON is built on the MOOSE Framework, it can accomodate many types of problems, and therefore does not expect dynamics problems only. Because this is a dynamics simulation, the user must define the [DynamicTensorMechanics](syntax/Kernels/DynamicTensorMechanics/index.md) block, along with the inertia blocks for the three directions. There is a large library of options for this block in the input file. See [Kernels](syntax/Kernels/index.md) for more information on the Kernels system.
+
+!listing tutorial/site_response_1.i
+         start=AuxKernels
+         end=BCs
+
+The AuxKernels block is defined next. This block mimics the Kernels block, but AuxVariables act on some variable and can be defined explicitly with known functions. Here is where we setup all acceleration and velocity variables, this is also where stress or other field variables would be set. See [AuxKernels](syntax/AuxKernels/index.md) for more information on the AuxKernels system.
+
+!listing tutorial/site_response_1.i
+         start=BCs
+         end=Functions
+
+Now that all of the variables have been defined and set up properly, we can begin to formulate the problem for the analysis. The BCs block is where problem boundary conditions are applied. We use the PresetBC type to fix the bottom surface of the soil column in the Y and Z directions, then prescribe an acceleration history to the bottom surface using the PresetAcceleration type. This is defined in the X direction only, for this problem. In order to constrain the soil column so that it only moves in shear, a periodic boundary condition is applied along the side surfaces of the column. This requires the user to define parallel surfaces along with a translational vector between those surfaces. See [BCs](syntax/BCs/index.md) for more information on the boundary conditions system.
```

Comment:
... periodic boundary condition is applied along the side surfaces of the soil column using the `Periodic` block. The side surfaces are input as boundaries. The boundaries called 'back', 'front', 'bottom', and 'top' are automatically created by the GeneratedMesh above. Usually, these boundaries will have to be defined in Cubit, Trelis, or any other meshing software being used. The Periodic BC requires the surfaces (input as boundaries) to be perfectly parallel and the normal vector of translation between the surfaces is input as the 'translation' parameter.  See [BCs](syntax/BCs/index.md) for more information on the boundary conditions system including the Periodic BC.